app/controllers/planning_applications_controller.rb

```diff
@@ -109,6 +148,12 @@ def unpermitted_status_for_user?(status)
     status == :awaiting_determination if current_user.assessor?
   end
 
+  def document_validation_params
+    params.require(:planning_application).permit(:status,  :'documents_validated_at(3i)',
+                                                 :'documents_validated_at(2i)',
+                                                 :'documents_validated_at(1i)')
```

Comment:
We're not doing any bulk attributes, so don't need to use strong params, and I'm a bit confused by the inclusion of `status` in here. Won't `document_validation_params.values.join("-")` = 'not_started-31-12-2020'? Is that why the `match` is used further above?

Reply:
`:status` was a regression that I left in by accident. I guess maybe a better approach would be to define and join the three date field params in a method other than using strong params, as we need to just define them rather than explicitly permit them? Unless you can think of a better way, which I'm sure there is.

Reply:
Yup, I agree, I think a method along the lines of `date_string_from_params` would probably be neater :-)